packages/back-end/src/events/handlers/discord/discord-event-handler-utils.ts

```diff
@@ -0,0 +1,580 @@
+import {
+  NotificationEvent,
+  LegacyNotificationEvent,
+} from "back-end/src/events/notification-events";
+import { APP_ORIGIN } from "back-end/src/util/secrets";
+import { ExperimentWarningNotificationPayload } from "back-end/src/validators/experiment-warnings";
+import { ExperimentInfoSignificancePayload } from "back-end/src/validators/experiment-info";
+import { ExperimentDecisionNotificationPayload } from "back-end/src/validators/experiment-decision";
+import {
+  SafeRolloutDecisionNotificationPayload,
+  SafeRolloutUnhealthyNotificationPayload,
+} from "back-end/src/validators/safe-rollout-notifications";
+
+export type DiscordEmbed = {
```

Comment:
Might be worth adding [discord-api-types](https://www.npmjs.com/package/discord-api-types) as dev dependency
[RESTPostAPIWebhookWithTokenJSONBody](https://discord-api-types.dev/api/discord-api-types-v10/interface/RESTPostAPIWebhookWithTokenJSONBody)